ludwig/hyperopt/utils.py

```diff
@@ -174,17 +174,29 @@ def substitute_parameters(
     return config
 
 
+@DeveloperAPI
+def contains_grid_search_parameters(config: ModelConfigDict) -> bool:
```

Comment:
Is this `ModelConfigDict` or `HyperoptConfigDict`? I think it should be the latter.

Reply:
Ah copy pasta + GitHub copilot, yeah should be ModelConfigDict.